hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/schemas/V053AddressBookSchema.java

```diff
@@ -63,6 +64,9 @@ public void migrate(@NonNull final MigrationContext ctx) {
                 ctx.newStates().get(NODES_KEY);
         final var networkInfo = ctx.networkInfo();
         final var addressBook = networkInfo.addressBook();
+        final var bootstrapConfig = ctx.configuration().getConfigData(BootstrapConfig.class);
+        final var adminKey =
+                Key.newBuilder().ed25519(bootstrapConfig.genesisPublicKey()).build();
```

Comment:
Is this discussed. Setting genesis Key on all nodes ?

Reply:
Requirements is to use Account 0.0.55's key as adminKey. From com.hedera.node.app.service.token.impl.schemas.SyntheticAccountCreator#generateSyntheticAccounts, you can see we use this key, superUserKey as 0.0.55 key.

Reply:
Please forgive my possibly naive question.
Is this method called at genesis only, or is this possibly called long after genesis?  I ask because account `0.0.55` might _change keys_ after genesis, and (I would assume) might no longer match the value in bootstrap configuration.

I just wonder if we should read the actual key on the account rather than assuming it hasn't changed?

Reply:
All system accounts use this superKey.

Reply:
And they never change it/cannot change it?

Reply:
Add this to tomorrow's meeting topic. Not using the account here is because the account might not have been created at this point.